spec/requests/api/v1/relationships/items/items_invoice_items_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe 'Items invoice items relationship' do
+  it 'returns collection of associated invoice items' do
+    item = create(:item)
+    create_list(:invoice_item, 100, item_id: item.id)
```

Comment:
I think this test would be better if we created a few more items, with their own list of invoices, so we can make sure we are not just getting all invoice_items.